website/docs/d/scc_posture_profiles.html.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_profiles"
+description: |-
+  Get information about list_profiles
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_profiles
+
+Review information about the security and compliance center posture latest scans. For more information, about profiles see https://cloud.ibm.com/docs/security-compliance?topic=security-compliance-profiles.
```

Comment:
please do provide the hyperlink using []() syntax as shown in above, and should not provide stage docs link for the registry docs.